### PR TITLE
Fix various spelling fixes.

### PR DESCRIPTION
**Pull request by jmallach**

### CODE REVIEW

**Comment by jmallach** [comment]
Please be careful with a few that maybe touch labels that can't be touched. In particular, I am not sure about "avaliable".

File: hash/amigaocs_flop.xml [hunk 1]
```diff
@@ -28534,7 +28534,7 @@
 
 	<software name="millen" supported="no">
 		<!-- SPS (CAPS) release 2027 -->
-		<description>Millenium - Return to Earth (USA)</description>
+		<description>Millennium - Return to Earth (USA)</description>
```
Comment (cuavas) [inline hash/amigaocs_flop.xml]: This is a software title - spelling errors should be preserved.

File: src/mame/layout/j2lucky2.lay [hunk 2]
```diff
@@ -206,7 +206,7 @@
 		<rect state="0">
 			<color red="0.25" green="0.25" blue="0.00"/>
 		</rect>
-		<text string="When avaliable after ">
+		<text string="When available after ">
```
Comment (cuavas) [inline src/mame/layout/j2lucky2.lay]: Please provide a photo of the machine showing that this is not an original spelling error.

File: src/mame/layout/j2lucky2.lay [hunk 3]
```diff
@@ -246,7 +246,7 @@
 			<color red="0.0" green="0.0" blue="0.0"/>
 			<bounds x="0.05" y="0.05" width="0.90" height="0.45"/>
 		</text>
-		<text string="AVALIABLE">
+		<text string="AVAILABLE">
```
Comment (cuavas) [inline src/mame/layout/j2lucky2.lay]: Same with this, show evidence it isn't an original error.

File: hash/amigaocs_flop.xml (hunk 1 above)
Comment (startaq) [inline hash/amigaocs_flop.xml]: The game is called "Millenium - Return to Earth". Yes, it's spelled incorrectly. See http://hol.abime.net/5159.

**Comment by jmallach** [comment]
Thanks for the comments. As discussed on IRC, I looked in the Wikipedia for the Return to Earth case and learned it's definitely not the best source for this.

**Pull request merged**